CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ project(crypto C)
 option(CODECOV "Code Coverage" OFF)
 option(CRYPTO_LIBGCRYPT "Cryptography Module - Libgcrypt" ON)
 option(CRYPTO_KMC "Cryptography Module - KMC" OFF)
-option(DEBUG "Debug" OFF)
+option(DEBUG "Debug" ON)
```

Comment:
Should we leave debug off by default?

Reply:
Yep - just missed it. Good catch. Will fix.